### DISCUSSION TITLE
droop control implementation

### DISCUSSION THREAD

**Discussion by BinHuangScut** [e2nIEE/pandapower#1282] [Q&A]
Hi all,
Thank you for your efforts on pandapower. This is an amazing package.
I would like to ask if you have any plans to release the code related to droop control in the near future, or if you can share the test code of droop control implemented by control module.
My own implementation cannot work with the time-series simulation module. However, I would like to simulate the quasi-steady load flow of  the distribution system with droop controlled PV generators. 
Any help is much appreciated.

Thank you,
Bin

**Comment by rbolgaryn** [comment]
Dear @BinHuangScut ,

maybe as a quick solution, you can apply the CharacteristicControl to your use-case.

Best regards, 
Roman
